### PR TITLE
Prepare for 0.1.0 release

### PR DESCRIPTION
**Pull request by mocsharp**
### Description
- Update user guide for 0.1.0 release
- Update changelog for release

### Status
**Work in progress**

### Types of changes

- [ ] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] All tests passed locally by running `./src/run-tests-in-docker.sh`.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.
- [ ] User guide updated.
- [ ] I have updated the changelog

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+An [EditorConfig](https://editorconfig.org "EditorConfig homepage") file (`.editorconfig`) has been provided at the root of the runtime repository, enabling C# auto-formatting conforming to the above guidelines.
+
+
+##### License information
+
+All source code files should start with this paragraph:
+
+```
+// Copyright 2021-2022 MONAI Consortium
```
Comment (evberrypi) [inline CONTRIBUTING.md:76]: Developers are going to C&P this into every file, even if new.
New files should only have the year they were created, eg Copyright 2022. A file that was created in 2021 and modified this year would read Copyright 2021-2022. 
My suggestion would be to templatize the year in this file:  <<YEAR>>

**Review by evberrypi — CHANGES_REQUESTED** [review]
One comment on licensing

File: CONTRIBUTING.md (hunk 1 above)
Comment (mocsharp) [reply CONTRIBUTING.md]: Updated

**Review by evberrypi — APPROVED** [review]

**Comment by dbericat** [comment]
Approved!

**Pull request merged**